docs/ocis/development/testing.md

```diff
@@ -376,6 +376,48 @@ BEHAT_FEATURE="tests/acceptance/features/apiNotification/emailNotification.featu
 make test-acceptance-api
 ```
 
+## Running Test Suite With Tika Service (@tikaServiceNeeded)
+
+Test suites that are tagged with `@tikaServiceNeeded` require tika service.
+
+### Setup Tika Service
+
+Run the following docker command to setup tika service
+
+```bash
+docker run -d -p 127.0.0.1:9998:9998 apache/tika
+```
+
+### Run oCIS
+
+Documentation related to the content based search and tika extractor can be found [here](https://doc.owncloud.com/ocis/next/deployment/services/s-list/search.html#content-extraction)
+
+```bash
+# init oCIS
+IDM_ADMIN_PASSWORD=admin \
+ocis/bin/ocis init --insecure true
+
+# run oCIS
+PROXY_ENABLE_BASIC_AUTH=true \
+OCIS_INSECURE=true \
+SEARCH_EXTRACTOR_TYPE=tika \
+SEARCH_EXTRACTOR_TIKA_TIKA_URL=http://localhost:9998 \
+SEARCH_EXTRACTOR_CS3SOURCE_INSECURE=true \
+ocis/bin/ocis server
+```
+
+### Run the Acceptance Test
+
+Run the acceptance test with the following command:
+
+```bash
+TEST_WITH_GRAPH_API=true \
+TEST_OCIS=true \
```

Comment:
do we need use it? I should probably get rid of it all over the place

Reply:
Sure i will create another issue for it.

Reply:
I think it should be the part of this issue https://github.com/owncloud/ocis/issues/6998 which is in progress